international-development-policy-journal.csl

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>International Development Policy Journal</title>
```

Comment:
@cathbrend, could you change the title to `<title>International Development Policy</title>`? ("Journal" doesn't seem to be part of the journal title; https://poldev.revues.org/)

Please also update the style's file name (https://help.github.com/articles/renaming-a-file/), ID, and "self" link.